docs/generated/errors-list.md

```diff
@@ -214,7 +214,14 @@ The `galasactl` tool can generate the following errors:
 - GAL1215E: An attempt to update a user '{}' failed. Unexpected http status code {} received from the server. Error details from the server are not in a valid json format. Cause: '{}'
 - GAL1216E: An attempt to update a user '{}' failed. Unexpected http status code {} received from the server. Error details from the server are: '{}'
 - GAL1217E: An attempt to update a user '{}' failed. Unexpected http status code {} received from the server. Error details from the server are not in the json format.
-- GAL1225E: Failed to open file '{}' cause: {}. Check that this file exists, and that you have read permissions.
+- GAL1218E: Failed to update a monitor named '{}'. Sending the put request to the Galasa service failed. Cause is {}
+- GAL1219E: Failed to update a monitor named '{}'. Unexpected http status code {} received from the server.
+- GAL1220E: Failed to update a monitor named '{}'. Unexpected http status code {} received from the server. Error details from the server could not be read. Cause: {}
+- GAL1221E: Failed to update a monitor named '{}'. Unexpected http status code {} received from the server. Error details from the server are not in a valid json format. Cause: '{}'
+- GAL1222E: Failed to update a monitor named '{}'. Unexpected http status code {} received from the server. Error details from the server are: '{}'
+- GAL1223E: Failed to update a monitor named '{}'. Unexpected http status code {} received from the server. Error details from the server are not in the json format.
+- GAL1224E: Galasa Monitor named {} is not known on the Galasa service.
+- GAL1225E: Invalid monitor name provided. The name provided with the --name flag cannot be empty and must only contain characters in the following ranges: 'a'-'z', 'A'-'Z', '0'-'9', '-' (dash), '_' (underscore).
```

Comment:
What is the name of the 'system' cleanup monitor ? Perhaps it should start with a '_system' or something ?